portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,28 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>hey it's malachi!</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+  <style>
+      img {
+  width:360px;
+  height:640px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+body {background-color: #D4E7FF;}
+</style>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+<h2><center>★ MALACHI BLENMAN'S PORTFOLIO ★</center></h2>
+<img src = "images/IMG_0101.PNG">
+<p><center>Hi, my name is <b>Malachi Blenman</b> and I am a Sophomore Computer Engineering Major at Binghamton University. I am based in Brooklyn, NY and I have an affinity for all things tech! In the future, I would like to be an Aerospace Engineer and Astronaut, and overall learn more about space studies!</center></p>
+<p><img src = "images/Meet the Author.PNG"></p>
```

Comment:
Does this image work? I can't test it myself, but think you might need to replace the spaces in the url with `%20` which is just the http url "encoding" for a space.